src/app/cash-contracts.service.ts

```diff
@@ -138,48 +142,39 @@ export class CashContractsService {
     this.isSecretInStorageSubject.next(cc.Wallet.isSecretInStorage());
   };
 
-  createBuyOffer = async (utxo: cc.UtxoEntry, params: cc.TradeOfferParams) => {
-    const offer = cc.acceptTradeOfferTx(this.wallet, utxo, params);
+  createBuyOffer = async (utxo: cc.UtxoEntry, params: cc.TradeOfferParams, tokenDetails: {decimals: number}) => {
```

Comment:
`tokenDetails: {decimals: number}`

The { decimals: number } typing should to be set to the real type 😊